docs/reference/functions/typed.md

```diff
@@ -23,7 +23,7 @@ math.typed(signatures) : function
 Parameter | Type | Description
 --------- | ---- | -----------
 `name` | string | Optional name for the typed-function
-`signatures` | Object&lt;string, function&gt; | Object with one ore multiple function signatures
+`signatures` | Object&lt;string, function&gt; | Object with one or multiple function signatures
```

Comment:
Would it read better to have `one or more function signatures`?

Reply:
@harrysarson I'm not sure, I'm not a native speaker...